articles/TOC.yml

```diff
@@ -12,25 +10,31 @@
     href: install-guide/update.md  
   - name: Write your first quantum program
     href: quickstart.md
-- name: Managing quantum machines and drivers
-  href: machines/toc.yml
-- name: Quantum development techniques
-  topicHref: techniques/index.md
-  href: techniques/toc.yml
-- name: Q# libraries
-  href: libraries/toc.yml
-- name: Q# programming language
-  topicHref: language/index.md
-  href: language/toc.yml
-- name: Contributing to the Quantum Development Kit
-  href: contributing/toc.yml
-  topicHref: contributing/index.md
-- name: Glossary
-  href: Glossary.md
-- name: For more information
-  href: for-more-info.md
-- name: Release Notes
-  topicHref: relnotes/index.md
-  href: relnotes/toc.yml
-- name: License
-  href: license.md
+- name: Concepts
```

Comment:
I'd suggest moving the techniques chapter above the QC concepts chapter, if only to help address the issue feedback we've gotten that the concepts guide currently feels like an intimidating prereq.